doc/develop/manifest/external/cannectivity.rst

```diff
@@ -0,0 +1,57 @@
+.. _external_module_cannectivity:
+
+CANnectivity USB to CAN adapter firmware
+########################################
+
+Introduction
+************
+
+`CANnectivity`_ is an open source firmware for Universal Serial Bus (USB) to Controller Area Network
+(CAN) adapters.
+
+The firmware implements the Geschwister Schneider USB/CAN device protocol (often referred to as
+"gs\_usb"). This protocol is supported by the Linux kernel SocketCAN `gs\_usb driver`_, by
```

Comment:
do you really need to escape underscores? I am assuming you did this because you encoutered some issues, but this looks weird?

Reply:
Yes, otherwise they are considered referenced to "gs" despite the lack of grouping backticks. This is done in our existing documentation as well.

Reply:
Mmmmh you had me test because this *really* confused me :) I just removed the 4 backslashes from your change and it works just fine. I took the liberty to push to your branch. Please let me know if you still see issues, as this would be a bug IMO

Reply:
@henrikbrixandersen CI just completed and unless I'm missing something, it's all good withouth the escaped underscores, right?

Reply:
Looks like it, yes, Building locally now. I had so many issues with this this morning, cryptic error messages from templates and what not. Finally tracking it down to escaping of the underscores, which solved it.

Reply:
After updating the Python packages locally (`pip install -U -r doc/requirements.txt`) I can no longer reproduce the failure with the unescaped underscores.

Reply:
Which was probably a good thing to do anyway :) Should you see those cryptic error messages again, please feel free to hit me up!

Reply:
Thanks.